### PR TITLE
Enable Continuous Integration for pytorch/xla

### PR DESCRIPTION
**Pull request by jysohn23**

### CODE REVIEW

File: kokoro_build.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+# Update compilers to use gcc/g++/cpp-6 as default
+sudo apt-get update
+sudo apt-get install -y gcc-6
```
Comment (dlibenzi) [inline kokoro_build.sh]: GCC 6 is fine?
I was getting ICE with 6.3.x

Comment (jysohn23) [reply kokoro_build.sh]: Yes it's fine for Ubuntu, which is used by Kokoro. We can change to GCC 7 if you'd prefer.

Comment (dlibenzi) [reply kokoro_build.sh]: If 6 works we can keep it there for this case.

Comment (jysohn23) [reply kokoro_build.sh]: Sounds good.

**Pull request merged**